sqlvm-alwayson-cluster/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Create a AlwaysOn Deployment",
```

Comment:
Might want to put in the terms "SQL" and "cluster" in here so it's easy searchable